components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -422,6 +426,9 @@ struct Integrate {
         conflicts_with = "use-on-disk-deployment-plan"
     )]
     pub use_computed_deployment_plan: bool,
+    /// Path to Package.json
+    #[clap(long = "package")]
```

Comment:
It should probably conflict with the two others properties right (it shouldn't be possible to specify a package as well as manifest_path or use_computed_deployment_plan)?

Reply:
True! nice catch

Reply:
I wonder too about the description. Could be better.. Any idea?